### PR TITLE
Prajakta

### PR DESCRIPTION
**Pull request by patilprajakta**
Boilerplate code

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,85 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.example.synerzip.snapxeats"
+        minSdkVersion 15
+        targetSdkVersion 27
+        versionCode 1
+        versionName "1.0"
```
Comment (trivedidurgesh) [inline app/build.gradle]: change version no to 0.1 .

File: app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,85 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.example.synerzip.snapxeats"
+        minSdkVersion 15
```
Comment (trivedidurgesh) [inline app/build.gradle]: use minSdkVersion 21 .

File: app/build.gradle (hunk 1 above)
Comment (patilprajakta) [reply app/build.gradle]: Changes done

File: app/build.gradle (hunk 2 above)
Comment (patilprajakta) [reply app/build.gradle]: Done

**Comment by trivedidurgesh** [comment]
Looks good

**Pull request merged**